### ISSUE TITLE
[Feat]: recognize new albums via deezer directly

### ISSUE THREAD

**Issue by 1337sup3rh4x0r** [RandomNinjaAtk/docker-lidarr-extended#152]
Lidarr relies on musicbrainz data (which is great!! we all love FLOSS!!) that is sometimes a bit slower to reflect new albums, especially for less-known artists.

Would it be possible for lidarr-extended to include an option that directly downloads all albums as they become available on deezer (or maybe configurably also singles) of artists that are already monitored in lidarr and copies/imports them to the folder structure even before lidarr becomes aware of these albums via musicbrainz?

Basically that is what I do manually currently. I have a separate deemix container which i use to manually grab those albums and then import them via beets. That could also be automated here, couldn't it? :)

I am not sure if this can be achieved easily by making use of API of deemix but I am sure you have an opinion on the feasibility of this.

Thank you for this project!

**Comment by RandomNinjaAtk** [comment]
No, the goal is for Lidarr to manage the data. Previous projects I had created did something to that affect, but that is not the goal of this project and will not be implemented. My suggestion to resolve this would be to become an active part of the community and help maintain/update Musicbrainz, that way it is there and then these scripts/lidarr can do their job as intended.

**Comment by 1337sup3rh4x0r** [comment]
You are absolutely right. Of course this is the ideal path to pursue. And there are tools like a-tisket (https://atisket.pulsewidth.org.uk/) that make it easier. Still it requires quite some manual involvement.

I know this might be getting a bit OT now, but since you seem to be into this topic:
Do you know an easy way to compare ones own collection against musicbrainz and identify missing albums that one could contribute to musicbrainz? Maybe some beets plugin?
